### PR TITLE
Add changes made in this comment: https://github.com/Kong/insomnia/pu…

### PR DESCRIPTION
**Pull request by wongstein**
…ll/3055#issuecomment-871148003

### Summary

This PR adds changes made to inso-cli optional commands, discussed here in this comment:

https://github.com/Kong/insomnia/pull/3055#issuecomment-871148003

### Reason

Several changes to inso-cli have been made related to git support story, and inso-cli docs need to reflect those changes

### CODE REVIEW

File: docs/inso-cli/cli-command-reference/inso-export-spec.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ inso export spec [identifier]
 Option  | Alias | Description
 ----- | ------ | -------
 `--output <path>` | -o | Save the specification to a file in the working directory.
+`--workingDir <path>` || Specifying the current working directory.
```
Comment (lena-larionova) [inline docs/inso-cli/cli-command-reference/inso-export-spec.md:24]:
```suggestion
`--workingDir <path>` || Specify the current working directory.
```
Is there a default? Is the default just `$HOME`? I'd add that to the description.

Comment (wongstein) [reply docs/inso-cli/cli-command-reference/inso-export-spec.md:24]: Oh I agree that wording is confusing.  Without --workingDir, the default is to use the current folder the program is running in as the "working directory".  When you set --workingDir <path>, then the program will look at the folder in <path> and use that as the working directory.

I guess the analogy is:  I tell you to cook an egg in the kitchen.  You'll, by default, think the kitchen is the one where you are now.  If I say "Cook an egg in the kitchen in my house", now you know to not cook an egg wherever you are now, but to come to my house and my kitchen and cook it.

Did that make sense?

**Review by lena-larionova — APPROVED** [review]
Changes for this file LGTM.

Looks like these flags are global though, will you be making changes in every inso command reference file?

**Comment by wongstein** [comment]
Ah, @lena-larionova good catch (on the global flag point).  Would it be better to move the options to here:
https://insomnia-docs-o0ybx3vnp-teaminsomnia.vercel.app/inso-cli/cli-command-reference

And maybe have it also in all the other command files?

**Comment by lena-larionova** [comment]
> Would it be better to move the options to here:
https://insomnia-docs-o0ybx3vnp-teaminsomnia.vercel.app/inso-cli/cli-command-reference

Yeah, I think so. Would be clearer.

> And maybe have it also in all the other command files?

+1 to this too - maybe with a section titled something like "Global flags". Kind of like this: https://docs.konghq.com/deck/latest/reference/deck_diff/

**Comment by wongstein** [comment]
@lena-larionova Updated with latest and greatest!

**Pull request merged**